.github/workflows/python_sdk_unit_tests.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
     steps:
       - name: Check out code
-        uses: actions/checkout@v3
+        uses: 4
```

Comment:
Is this right? Shouldn't be this `actions/checkout@v4`.

Reply:
fixed